src/train.py

```diff
@@ -0,0 +1,67 @@
+# pylint: disable=too-many-locals
```

Comment:
You can remove this PyLint disable, I disabled it for this project in the pylint config file